build.sbt

```diff
@@ -32,6 +32,13 @@ val nailgun = project
     hijackScalafmtOnCompile in SbtConfig in NailgunBuild := false,
   )
 
+val compilerBenchmark = project
```

Comment:
I propose we call this benchmark bridge.

Reply:
I've changed it